src/CftfBundle/Resources/views/LsDoc/index.html.twig

```diff
@@ -43,10 +43,8 @@
                     <div class="col-md-12">
                         ASN:
                         <input name="import" type="radio" value="asn" checked="checked"></input>
-                        {% if github_client_id is not empty %}
```

Comment:
Was this removed on purpose?

If the specific instance of Salt does not have a github_client_id set in the parameters file (which also requires it to be registered with GitHub) then there is no way any of the github importing can work.